src/Client.ts

```diff
@@ -114,6 +115,7 @@ export type ClientOptions = NetworkOptions & KeyStoreOptions & ContentOptions
 export function defaultOptions(opts?: Partial<ClientOptions>): ClientOptions {
   const _defaultOptions: ClientOptions = {
     keyStoreType: KeyStoreType.networkTopicStoreV1,
+    privateKeyOverride: undefined,
```

Comment:
[PEBBLE] Long term I'm not in love with this pattern.

1) It does not provide guidance on how these keys ought to be protected. 
2) It does not provide a mechanism to safe guard these keys in the future.

What are your thoughts on repurposing the KeyStore object to better suit developers needs? The idea being that we could provide secure storage mechanisms that satisfy 90% of the use cases and facilitate developers building their own if needs be. 

If a "OfflineBroswerKeyStore" was available in the SDK which did most of the heavy lifting, there would be significantly less burden on developers. We would then be able to push out security patches and updates. 

Thoughts?

Reply:
Can you elaborate on what exactly the `OfflineBrowserKeyStore` would do?

Reply:
I don't think the two are mutually exclusive. But we should have some escape hatch for developers whose needs we can't currently serve with a bespoke approach. There are a lot of different use-cases here (server, mobile, extension, etc, etc)

Reply:
There's maybe a secondary question about whether we should express this as some sort of "StaticStore" instead of just a client option. I wouldn't be opposed to that, although the API doesn't quite line up with the other providers since it would always resolve to a single wallet and couldn't accept writes.

Reply:
I very much agree that we will need an escape hatch, but I also hope it is rarely used ( and if it is, then it can be shared as a package so others can benefit from it as well). Instead major use cases are covered by existing keyStores, and developers are incentivized to use those instead. 

Using a keyStore interface for supplying keys would help keep some consistency across the different approaches, while also enforcing a common interface.  Symmetry is the focus here.

The StaticStore, is exactly how I would think about the lowest level escape hatch. Developers could take full control, if really needed.

Reply:
I'm cognizant that this would require a fair amount of work to revamp our flow and define an interface, so I'm not putting a block on anything. But as I look forward I see the need for clients to handle keyRotations and ratcheting, which will require more than just passing in a private key.

Reply:
I reformatted this PR this to use a `StaticKeyStore`. It doesn't change much in the present, but puts us on better footing to make the system more full-featured down the line.

Couldn't agree more that I hope the escape hatch is very rarely used.